pkgs/development/tools/yarn-berry/default.nix

```diff
@@ -43,16 +47,18 @@ stdenv.mkDerivation (finalAttrs: {
   passthru.updateScript = ./update.sh;
 
   passthru.tests = {
-    version = testers.testVersion {
-      package = finalAttrs.finalPackage;
-    };
+    version = testers.testVersion { package = finalAttrs.finalPackage; };
```

Comment:
I think this change shouldn't be necessary too with https://github.com/NixOS/nixfmt/pull/224 .

Reply:
currently, the CI does not complain. Should we wait for an update there?

Reply:
> currently, the CI does not complain. Should we wait for an update there?

It won't complain either way, I'm just an advocate of minimal diffs, and an advocate of always putting `*Inputs` and other attributes, that very often grow/change in future updates of a package.

Reply:
I tried to apply your suggestions, but `nixfmt` always redid what is seen here. I prefer not to interfere with autoformating.

Reply:
> I tried to apply your suggestions, but `nixfmt` always redid what is seen here. I prefer not to interfere with autoformating.

Are you sure you are using the latest `nixfmt`? Note also the difference between `nixfmt-classic` and `nixfmt-rfc-style`.

Reply:
> yes.
> 
> ```
> nixfmt nixpkgs-unstable-2024-08-16
> ```
> 
> which is on current `nixos-unstable` and `master` (and yes thats `nixfmt-rfc-style`)

Not my experience:

https://asciinema.org/a/ynlyFqkX7cjaSOMBquT677rYi